cl/lib/zoho.py

```diff
@@ -0,0 +1,101 @@
+from django.conf import settings
+from django.core.cache import cache
+from zohocrmsdk.src.com.zoho.api.authenticator import OAuthToken
+from zohocrmsdk.src.com.zoho.crm.api import (
+    HeaderMap,
+    Initializer,
+    ParameterMap,
+)
+from zohocrmsdk.src.com.zoho.crm.api.record import (
+    APIException,
+    RecordOperations,
+    ResponseWrapper,
+    SearchRecordsParam,
+)
+
+
+def get_zoho_cache_key() -> str:
+    return "zoho_token"
+
+
+class SearchableModule:
+    module_name: str = ""
+
+    @staticmethod
+    def initialize():
+        refresh_token = cache.get(f"{get_zoho_cache_key()}:refresh")
+        if not refresh_token:
+            raise Exception(
+                f"Cache miss: no value found for key {get_zoho_cache_key()}:refresh. "
+                "Please run `cl_get_zoho_tokens` to refresh and store new tokens."
+            )
+        token = OAuthToken(
+            client_id=settings.ZOHO_CLIENT_ID,
+            client_secret=settings.ZOHO_CLIENT_SECRET,
+            refresh_token=refresh_token,
+        )
+        Initializer.initialize(
+            environment=settings.ZOHO_ENV,
+            token=token,
+            store=settings.ZOHO_STORE,
+            resource_path=settings.ZOHO_RESOURCE_PATH,
+        )
+
```

Comment:
Would it be a good idea to automatically call `initialize()` when instantiating the class?
When I was testing it, I ran into an error because I forgot to call `initialize()` first.

Maybe we could also raise an error if `self.module` is not defined?

```
def __init__(self) -> None:
        if Initializer.get_initializer() is None:
            self.initialize()

        if not self.module_name:
            raise Exception(
                "Subclasses must set `module_name`."
            )
```